### ISSUE TITLE
Blank Front End

### ISSUE THREAD

**Issue by pool2mine** [phucvinh52/open-ethereum-pool#1]
Hello @phucvinh52 i see you updated a little bit the software. I tried to build with node 4.x and got some errors. After i updated to node 6.x everything was working good, but still getting a blank front end just with background and footer. something is happening and i am not able to see stats on the front end. can you help me with this? thanks!

**Comment by phucvinh52** [comment]
can you inspect in browser and screenshot to me :)

**Comment by pool2mine** [comment]
Hi, Sure!
![oep](https://user-images.githubusercontent.com/38216354/62403677-0f03bd00-b55d-11e9-8e67-4bf71629d595.png)

**Comment by pool2mine** [comment]
@phucvinh52 I had also same issue with sammy007 version. I'm able to build, i have some warnings but no errors. When i stop the pool i get the "Stats API Temporarily Down" which is correct, but when i start the pool again the home page goes blank, no stats at all, just background and footer is showing

**Comment by phucvinh52** [comment]
> @phucvinh52 I had also same issue with sammy007 version. I'm able to build, i have some warnings but no errors. When i stop the pool i get the "Stats API Temporarily Down" which is correct, but when i start the pool again the home page goes blank, no stats at all, just background and footer is showing

give me your file https://github.com/phucvinh52/open-ethereum-pool/blob/master/www/config/environment.js
If default you need replace something in this file

**Comment by pool2mine** [comment]
I only replaced my domain name on the 3 lines needed (API, HTTP and Stratum)


    APP: {
      // API host and port
      ApiUrl: '//my.test.domain.com/',

      // HTTP mining endpoint
      HttpHost: 'http://my.test.domain.com',
      HttpPort: 8888,

      // Stratum mining endpoint
      StratumHost: 'my.test.domain.com',
      StratumPort: 8008,

I only changed that... the rest is intact.

**Comment by 1blockologist** [comment]
were you able to resolve this?

were you able to find instructions to submit work and earn blocks without the gui?

**Comment by pool2mine** [comment]
hi after several months i finally saw a missing dependency in the node_modules folder, but i think your question is different, the block unlocker and api are 2 separate processes... you don't need api in order for your pool to work. api config is just for the website to show stats

**Comment by 1blockologist** [comment]
@pool2mine yeah, I'd like the site to show stats but I'll figure that out later

was the missing dependency still missing in this repository's package.json?

are you running geth or parity? if so what versions. all the latest software is kind of broken - like sammy's pool

**Comment by pool2mine** [comment]
I'm running the last version of geth. you're right, some repos are broken, but i was lucky i built the site in another test server before it was broken so i just built the new one in my running server and just copied and pasted the missing files. i tried to attach the folder here, but github does not allow rar files, if you contact me on the pool telegram chat i can send it to you.
